File: day 1.py
#include <stdio.h>

int main() {
    int n, sum = 0;

    printf("Enter the value of N: ");
    scanf("%d", &n);

    for(int i = 1; i <= n; i++) {
        sum = sum + i;
    }

    printf("Sum of first %d natural numbers = %d", n, sum);

    return 0;
}
q2.c
#include <stdio.h>

int main() {
    int n;

    printf("Enter a number: ");
    scanf("%d", &n);

    printf("Multiplication Table of %d:\n", n);

    for(int i = 1; i <= 10; i++) {
        printf("%d x %d = %d\n", n, i, n * i);
    }

    return 0;
}

File: day 1.cpp
q1.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, sum=0;
    cout<<"enter the value of n:";
    cin>>n;
    for(int i=1;  i<=n; i++)
    {
        sum+=i;
        
    }
    cout<<"sum of first "<<n<<" natural numbers is: "<<sum;
    return 0;

}

q2.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<<"enter a number:";
    cin>>n;
    for (int i=1; i<=10; i++)
    {
        cout<<n<<" * "<<i<<" = "<<n*i<<endl;
    }
    return 0;
}

q3.cpp
#include<iostream>
using namespace std;
int main()
{
    int n;
    long long factorial=1;
    cout<<"enter a number:";
    cin>> n;
    for(int i = 1; i<=n; i++)
    {
        factorial *= i;
        
    }
    cout<<"factorial of "<<n<<" is: "<<factorial;
    return 0;
}

q4.cpp
#include<iostream>
using namespace std;
int main()
{
    int num,count = 0;
    cout<<"enter a number: ";
    cin >> num;
    while ( num!= 0) {
        num = num / 10;
        count++;
    }
    cout<<"Number of digits = " << count;
    return 0;
    }
